apps/betterangels-backend/accounts/tests/test_mutations.py

```diff
@@ -65,8 +86,8 @@ def test_delete_current_user(self) -> None:
         """
 
         response = self.execute_graphql(mutation)["data"]["deleteCurrentUser"]
-        self.assertEqual(response["id"], user.pk)
-        self.assertEqual(initial_user_count, User.objects.count())
+        self.assertEqual(response["id"], self.user.pk)
+        self.assertEqual(initial_user_count - 1, User.objects.count())
```

Comment:
nitpick: prefer this flipped
```suggestion
        self.assertEqual(User.objects.count(), initial_user_count - 1)
```